src/api/box-edit/BrowserUtils.js

```diff
@@ -0,0 +1,298 @@
+// @flow
+const BROWSER_CONSTANTS = {
```

Comment:
This file should ideally be inside util folder, and merged with Browser.js

Reply:
agreed, will do.

Reply:
I don't know if we want to bring this model into our browser utils. The box edit version passes an instance around and keeps track of state while we rely on static methods. We can refactor their utils but at that point we would own it.